cli/src/alluxio.org/cli/processes/local.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package processes
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+)
+
+var Local = &LocalProcess{
+	BaseProcess: &env.BaseProcess{
+		Name: "local",
+	},
+}
+
+var localProcesses = []env.Process{
```

Comment:
similar to the All command, define the list of processes to launch locally as a struct field rather than a package level variable